.travis.yml

```diff
@@ -219,16 +229,20 @@ install:
     - if [[ -n "$LATEST" ]]; then BUILD_COMMIT=master; fi
     - clean_code $REPO_DIR $BUILD_COMMIT
     - build_wheel $REPO_DIR $PLAT
+    - ls -l "${TRAVIS_BUILD_DIR}/wheelhouse/"
 
 script:
     - install_run $PLAT
 
-after_success:
-    # Upload wheels to Rackspace container
-    - if [[ -z "$LATEST" ]]; then
-        pip install wheelhouse-uploader;
-        python -m wheelhouse_uploader upload --local-folder
-          ${TRAVIS_BUILD_DIR}/wheelhouse/
-          --no-update-index
-          wheels;
-      fi
+# Upload wheels to GitHub Releases
+deploy:
+  provider: releases
+  api_key:
+    secure: PTgVG7DrYa2FTSQOq0eDaHDZb1vy0vf6MulyuoXMg8rssPQgJ/mYxRpNDK4V0EKolpN7f8s/OGg+fpNNtp5pOCJGsx0Okcf+YB2ac+Xl7DQPBucbDKFXs1ndf/ny6umk0TXX8JTrDp/mJDJf401yx1+qsZ6X/PFvchXvXVrQ+SQ=
```

Comment:
This is an encrypted OAuth token created using `travis setup releases` on my account when inside a local checkout of the https://github.com/python-pillow/pillow-wheels/ repo (ie. not my fork).

https://docs.travis-ci.com/user/deployment/releases/#authenticating-with-an-oauth-token